bc_obps/compliance/models/rls_configs/compliance_summary.py

```diff
@@ -0,0 +1,19 @@
+from compliance.enums import ComplianceTableNames
+from rls.enums import RlsRoles, RlsOperations
+from rls.utils.helpers import generate_rls_grants
+
+
+class Rls:
+    role_grants_mapping = {
+        RlsRoles.INDUSTRY_USER: [
+            RlsOperations.SELECT,
+            RlsOperations.INSERT,
+            RlsOperations.UPDATE,
+            RlsOperations.DELETE,
+        ],
+        RlsRoles.CAS_DIRECTOR: [RlsOperations.SELECT],
```

Comment:
Same comment here, the compliance summary data is created from a report version, and should be immutable after that